sklearn/cluster/tests/test_hierarchical.py

```diff
@@ -264,6 +273,14 @@ def test_agglomerative_clustering():
     assert_array_equal(clustering.labels_, clustering2.labels_)
 
 
+def test_agglomerative_clustering_mem_mapped():
```

Comment:
naming style remark: I think we tend to use "memory_mapped"  or "memmap"  and maybe "memmapped" but not "mem_mapped" and "mem_map".

```suggestion
def test_agglomerative_clustering_memory_mapped():
```

Reply:
I was hesitating for the proper wording. Thanks for clarifying it.